avm/res/storage/storage-account/main.bicep

```diff
@@ -69,10 +69,7 @@ param privateEndpoints privateEndpointType
 param managementPolicyRules array?
 
 @description('Required. Networks ACLs, this value contains IPs to whitelist and/or Subnet information. If in use, bypass needs to be supplied. For security reasons, it is recommended to set the DefaultAction Deny.')
-param networkAcls networkAclsType = {
-  bypass: 'AzureServices'
-  defaultAction: 'Deny'
-}
+param networkAcls networkAclsType?
```

Comment:
If made optional, the description is incorrect now. We should probably have a test for that...